dbt/include/snowflake/macros/adapters.sql

```diff
@@ -42,11 +42,27 @@
   {%- set secure = config.get('secure', default=false) -%}
   {%- set copy_grants = config.get('copy_grants', default=false) -%}
   {%- set sql_header = config.get('sql_header', none) -%}
+  {% set existing_columns = adapter.get_columns_in_relation(relation) | map(attribute="name") | list %}
```

Comment:
Do we need to get `existing_columns`?

Reply:
Nope, I think this is left over from a previous version I was testing out. Removing this shortly.